### ISSUE TITLE
Create alt text report to obtain baseline measurement for pre-validation

### ISSUE THREAD

**Issue by joagnitti** [department-of-veterans-affairs/va.gov-cms#16026]
## Description
The alt text image report will be summarized based on the quality of alt text according to the criteria of succinct, redundant, uses filename, and accuracy. We are conducting this audit because we will be updating the alt text section of the CMS and need to obtain a baseline measurement of quality prior to the launch. 

## Acceptance Criteria
- [x] Data for alt text quality has been summarized:
    - [x] Succinct?
    - [x] Redundant?
    - [x] Uses Filename
- [x] Report has been created with summarized stats 
- [x] Report has been shared with Product

**Comment by joagnitti** [comment]
@EWashb @laflannery (cc @BerniXiongA6): [Here's a link to the audit](https://docs.google.com/presentation/d/1eFHE0c-Uz2j99jB2jLnOpx5T91vzvoJC44yIyGhorIs/edit#slide=id.g299549f0021_0_21). I'll also post this on Slack for more visibility.
